packages/libs/registry-sdk/src/lib/schemas/app.ts

```diff
@@ -59,6 +59,9 @@ const app = z
       .optional(),
     redirectUris: z
       .array(z.string().url())
```

Comment:
Can we use the z.set() type? https://zod.dev/api?id=sets
I think it enforces uniqueness by it's base design without using .refine()